src/api/dependenciesConfig.js

```diff
@@ -99,6 +106,12 @@ export const mintPeriodParser: C.Parser<MintPeriod> = C.object({
   weight: C.number,
 });
 
+function checkWeightValid(x: number): number {
+  if (x > 1 || x < 0)
+    throw new Error(`must be a number between 0 and 1, got ${x}`);
```

Comment:
actually, having a weight greater than 1 is technically fine. if there's a single dep with weight 1, it gets half the total cred. if the weight were 2, it would get 2/3rds of the total cred.

i suggest just checking that the weight is a nonnegative number